main.go

```diff
@@ -69,49 +92,84 @@ func main() {
 		},
 		cli.StringFlag{
 			Name:        "image-storage-access-key",
+			Usage:       "Image storage access key",
 			EnvVar:      "IMAGE_STORAGE_ACCESS_KEY",
 			Destination: &config.ImageStorageAccessKey,
 			Required:    true,
 		},
 		cli.StringFlag{
 			Name:        "image-storage-secret-key",
+			Usage:       "Image storage secret key",
 			EnvVar:      "IMAGE_STORAGE_SECRET_KEY",
 			Destination: &config.ImageStorageSecretKey,
 			Required:    true,
 		},
 		cli.BoolFlag{
 			Name:        "skip-authentication",
 			EnvVar:      "SKIP_AUTHENTICATION",
-			Destination: &config.SkipAuthentication,
 			Usage:       "Define whether to skip auth login or not, default to false",
+			Destination: &config.SkipAuthentication,
 		},
 		cli.StringFlag{
 			Name:   "authentication-mode",
 			EnvVar: "HARVESTER_AUTHENTICATION_MODE",
 			Usage:  "Define authentication mode, kubernetesCredentials and localUser are supported, could config more than one mode, separated by comma",
 		},
+		cli.StringFlag{
+			Name:        "profile-listen-address",
+			Value:       "127.0.0.1:6060",
+			Usage:       "Address to listen on for profiling",
+			Destination: &profileAddress,
+		},
+	}
+	app.Action = func(c *cli.Context) error {
+		// enable profiler
+		if profileAddress != "" {
+			go func() {
+				log.Println(http.ListenAndServe(profileAddress, nil))
+			}()
+		}
```

Comment:
Looks like the profiler is always enabled. But in the chart, it is controlled by the debug flag.

Reply:
removed chart condition check of profiling.